ui-v2/src/components/ui/docs-link.tsx

```diff
@@ -0,0 +1,22 @@
+import { ExternalLinkIcon } from "lucide-react";
+
+import { Button } from "./button";
+
+// Continually add to the list
+type HrefDocs =
+	| "https://docs.prefect.io/latest/guides/variables/"
+	| "https://docs.prefect.io/v3/develop/task-runners";
```

Comment:
What if we gave human-readable names to the links like 'variables-guide' and 'task-runners-guide' and allow those names to be provided to the `DocsLink` component? That way we could get type safety, make reusing links easier, and encapsulate the actual links from parent components.

Reply:
sounds good.

I can do a record, like:

```
{ 'variables-guid': '"https://docs.prefect.io/latest/guides/variables'}
```

and we just need to pass in something like:
```
<DocsLink id="variables-guid" />
```

Reply:
Updated on latest push